### ISSUE TITLE
Not able to login 

### ISSUE THREAD

**Issue by ashiqYousuf** [LeetCode-OpenSource/vscode-leetcode#821]
## 🐛 Bug Report

There's a serious catch I found on Lintcode, whenever I try to login it doesnt  redirect me to my profile page or problems section, don't know what's wrong going with, please look into the issue and resolve it as early as possible

**Comment by YUUSUKE18** [comment]
@ashiqYousuf 

Hi, you mean you cannot login using third party such as cookie, github and Linkedin, right?

**Comment by Sayak-Mallick** [comment]
> @ashiqYousuf

Yes sir i am unable to login by any method

**Comment by ashiqYousuf** [comment]
Yes  there's a redirection problem there. Don't know why they ain't working
on this.Without this how can anyone be able to access the resources.

**Comment by OnlyblackTea** [comment]
Plus, there's no way to login when Github requires a verify code.

**Comment by ashiqYousuf** [comment]
Of course I'm unable to login on lintcode.

**Comment by ashiqYousuf** [comment]
Problem solved now...
